packages/styles/src/components/text-input/obfuscatedField.js

```diff
@@ -2,6 +2,10 @@ class ObfuscatedField {
   constructor(component) {
     this.input = component;
     this.btn = component.nextElementSibling;
+     
+    if (this.btn) {
+      this.btn.setAttribute('aria-pressed', false)
```

Comment:
aria-attributes should always be a string. So 'false' should be quoted.

Reply:
`setAttribute` will convert non-string values to strings.

Reply:
Yep, I totally forgot that. Thinking about the issues I've run into in JSX.